--- a/makefile
+++ b/makefile
@@ -30,6 +30,7 @@
 # For Linux
 linux:
 	$(MAKE) lpeg.so "DLLFLAGS = -shared -fPIC"
+	$(MAKE) liblpeg.a "DLLFLAGS = -shared -fPIC"
 
 # For Mac OS
 macosx:
@@ -37,6 +38,9 @@
 
 lpeg.so: $(FILES)
 	env $(CC) $(DLLFLAGS) $(LDFLAGS) $(FILES) -o lpeg.so
+liblpeg.a: $(FILES)
+#	env $(CC) -c $(DLLFLAGS) $(LDFLAGS) $(FILES) -o lpeg.o
+	env ar rcs liblpeg.a $(FILES)
 
 $(FILES): makefile
 
